pokepaylib/src/main/java/jp/pokepay/pokepaylib/BankAPI/Account/CreateAccountCpmToken.java

```diff
@@ -22,13 +23,13 @@
     public String accountId;
     public int scopes;
     public Integer expiresIn;
-    public String additionalInfo;
+    public Metadata key1;
```

Comment:
I think change then mapping the API scheme.

Reply:
`{"key1": "value1"}` is just an example, it does not mean that the key name `"key1"` will always be specified.
The following examples are all correct input.

```
$ curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer NPfjxa5TlELYockxBi973mTp4vMWciuMM6auD1ubZTNZ4mJUJmLFcZXMTUiuUWt8" https://api-dev.pokepay.jp/accounts/7e1f4c8a-7e00-4c04-9afc-443317af76bc/cpm -d '{"metadata":"{}"}'

Response:
{"cpm_token":"90000022SxOK010ODjYea_", ... ,"metadata":{}}

$ curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer NPfjxa5TlELYockxBi973mTp4vMWciuMM6auD1ubZTNZ4mJUJmLFcZXMTUiuUWt8" https://api-dev.pokepay.jp/accounts/7e1f4c8a-7e00-4c04-9afc-443317af76bc/cpm -d '{"metadata":"{\"foo\":\"bar\"}"}'

Response:
{"cpm_token":"90000022SxOK01DZzlEO_2", ... ,"metadata":{"foo":"bar"}}

$ curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer NPfjxa5TlELYockxBi973mTp4vMWciuMM6auD1ubZTNZ4mJUJmLFcZXMTUiuUWt8" https://api-dev.pokepay.jp/accounts/7e1f4c8a-7e00-4c04-9afc-443317af76bc/cpm -d '{"metadata":"{\"foo\":\"bar\",\"baz\":\"qux\"}"}'

Response:
{"cpm_token":"90000022SxOK01lXD-M9D6", ... ,"metadata":{"baz":"qux","foo":"bar"}}

$ curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer NPfjxa5TlELYockxBi973mTp4vMWciuMM6auD1ubZTNZ4mJUJmLFcZXMTUiuUWt8" https://api-dev.pokepay.jp/accounts/7e1f4c8a-7e00-4c04-9afc-443317af76bc/cpm -d '{"metadata":"{\"key1\":\"value1\",\"key2\":\"value2\"}"}'

Response:
{"cpm_token":"90000022SxOK015uVJp-95", ... ,"metadata":{"key2":"value2","key1":"value1"}}
```

Reply:
Also please check if it could handle multiple key object like `{ "key1": "value1", "key2": "value2", ...etc }`

NOTE: `{ "key1": "{"nestedKey1": "value1", "nestedKey2": "value2" }" }` will respond Error from the server

Reply:
If the key name doesn't specific, how can the endpoint parse it?